Sources/MapboxMaps/Annotations/ViewAnnotationOptions.swift

```diff
@@ -0,0 +1,89 @@
+import Foundation
+import UIKit
+
+public struct ViewAnnotationOptions: Hashable {
+    public var geometry: Geometry? {
+        didSet {
+            mbxGeometry = geometry.map({ MapboxCommon.Geometry.init($0) }) ?? nil
+        }
+    }
+    public var width: CGFloat?
+    public var height: CGFloat?
+    public var associatedFeatureId: String?
+    public var allowOverlap: Bool?
+    public var visible: Bool?
+    public var anchor: ViewAnnotationAnchor?
+    public var offsetX: CGFloat?
+    public var offsetY: CGFloat?
+    public var selected: Bool?
+
+    // Used for hashing and constructing objc value
+    fileprivate var mbxGeometry: MapboxCommon.Geometry?
```

Comment:
I don't think we should do this. `MapboxCommon.Geometry` doesn't implement a useful `-isEqual:` method, so we'd also need to write out `==` explicitly to exclude this stored property. Let's just omit `geometry` from the hash.

Reply:
That's a good point, but the reason why I was thinking to include it is because currently the geometry field is required when adding a view annotation, but the rest can be omitted. So I expect that most users will just use a `ViewAnnotationOptions` with only a geometry value, maybe it would be useful to consider it in the hash.
(But I'll remove it from now just for simplicity.)

Reply:
I agree with this decision. I'll make a note about adding geometry into the hash in the future.